src/components/SchemaTable/index.jsx

```diff
@@ -46,12 +46,79 @@ function SchemaTable({ schema }) {
   const leftRows = [];
   const rightRows = [];
   /**
-   * TODO: define renderArray method
-   *       add description comment
+   * Create a single normal row (left and right column each).
+   * The result is stored in an object format in order for the
+   * pushRow() method to easily access each left and right column
+   * of the single row and push them to leftRows and rightRows respectively.
    */
-  const renderArray = schemaInput => (
-    <React.Fragment>{schemaInput}</React.Fragment>
-  );
+  const createNormalRow = schemaInput => ({
+    leftRow: (
+      <NormalLeftRow
+        key={leftRows.length + 1}
+        schema={schemaInput}
+        classes={classes}
+      />
+    ),
+    rightRow: (
+      <NormalRightRow
+        key={rightRows.length + 1}
+        schema={schemaInput}
+        classes={classes}
+      />
+    ),
+  });
+  /**
+   * Create a row to close off an array or object type schema.
+   * The left row displays a closing bracket of the type while
+   * the right row only consists of blank line padding.
+   */
+  const createClosingRow = type => {
+    const typeSymbol = {
+      array: ']',
+      object: '}',
+    }[type];
+
+    return {
+      leftRow: (
+        <div className={classes.row}>
+          <p className={classes.line}>{typeSymbol}</p>
+        </div>
+      ),
+      rightRow: (
+        <div className={`${classes.row} ${classes.rightRow}`}>
```

Comment:
I find using the following helps sometimes:

```jsx
import classNames from 'classnames';

// ...
<div className={classNames(classes.row, classes.rightRow)}>...</div>

// if we want to conditionally add a class name, there's a neat way of doing this
// for example:
const shouldAddRightRow = false;
<div className={classNames(classes.row, { [classes.rightRow]: shouldAddRightRow })}>...</div>
```

Reply:
Thanks! Applied changes!